.eslintrc.js

```diff
@@ -17,7 +17,6 @@ module.exports = {
   rules: {
     'no-unused-vars': ['warn', { args: 'none', argsIgnorePattern: 'req|res|next|val' }],
     'prettier/prettier': ['error'],
-    'space-before-function-paren': ['error', 'always'],
```

Comment:
i personally like this rule, though it's not a big deal either way, but we could just warn instead. Was it making something impossible?

Reply:
There's actually a problem with this rule which Matthew might have run into.  `eslint` will let you configure it to ask for these spaces, but I discovered yesterday that `prettier` doesn't listen to `eslint` and will always insist that there's no space.  Unfortunately, there's no option to configure `prettier` to ask for spaces.  A fork of `prettier` exists which allows for these spaces, but it looks so unpopular that I'm not sure that it's a good idea to go for it.

Reply:
Yep it was because of the conflict between what the two linters wanted.